src/renderer/prefComponents/keybindings/key-input-dialog.vue

```diff
@@ -0,0 +1,237 @@
+<template>
+  <div class="key-input-dialog">
+    <div v-if="showKeyInputDialog" class="input-overlay"></div>
+    <!-- TODO(KeyBinding): The dialog is shown behind the overlay when `modal` background is enabled. -->
+    <el-dialog
```

Comment:
@Jocs I added a workaround for this, do you know the issue?

Reply:
I also don't know.